pkg/util/cephcmds.go

```diff
@@ -47,15 +47,14 @@ func ExecCommand(program string, args ...string) (stdout, stderr []byte, err err
 	return stdoutBuf.Bytes(), nil, nil
 }
 
-// cephStoragePoolSummary strongly typed JSON spec for osd ls pools output
-type cephStoragePoolSummary struct {
+// CephStoragePoolSummary strongly typed JSON spec for osd ls pools output
+type CephStoragePoolSummary struct {
```

Comment:
@ShyamsundarR why we need above change? afaict, its used internally here .

Reply:
static linters complained that this was bad style, hence fixed it up.

Reply:
We dont need to export a type unwantedly..

Reply:
@humblec if you don't fix it CI will  fail

Reply:
this function may get  called by other functions  in  different package  hence it's required

Reply:
Fixed.

Reply:
@Madhu-1 if it was used in other packages, this wanted to be "exported" before this PR itself, Isnt it ? :).

Reply:
Thanks for fixing it @ShyamsundarR